src/League/ServiceServiceProvider.php

```diff
@@ -42,6 +42,16 @@ public function register()
      */
     private function registerService(ServiceDefinition $definition)
     {
+        if ($definition->isFactory()) {
+            $service = $this->getContainer()->add(
```

Comment:
@tomphp 

After assigning `$service` a value here, it is no longer used, should we remove it?